projects/pastanaga-angular/src/lib/styles/_chips.scss

```diff
@@ -0,0 +1,32 @@
+@import "variables";
+
+.pa-chip {
+    display: inline-flex;
+    align-items: center;
+    background: $color-neutral-primary-lighter;
+    height: rhythm(2.5);
+    max-width: rhythm(33);
+    padding: 0 0 0 rhythm(.25);
+    border-radius: rhythm(3);
+    .pa-close-chip-button .pa-button:not(:disabled).pa-secondary.pa-basic {
+        &:hover {
+            background: $color-neutral-secondary-lighter;
+        }
+        &:active {
+            background: $color-neutral-secondary-light;
+        }
+        &.pa-keyboard-focus {
+            background: $color-neutral-secondary-lighter;
+        }
+    }
```

Comment:
to be checked with francisco: are those overrides really necessary?

Reply:
forgot about this comment: the background of the chip is the same as the background of the button so it make sense to override it. Just frustrating.